portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,376 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400&display=swap');
+* {
+  box-sizing: border-box;
 }
 
-#greeting-container {
+body, html {
+  background-color: #474e5d;
+  height: 100%;
+  margin: 0;
+  font-family: 'Comfortaa', cursive;
+}
+
+/*Home View with Background*/
+header{
+  height: 100%;
+  background: linear-gradient(rgba(12, 12, 12, 0.99),rgba(12, 12, 12, 0.3)), 
+      url("/images/sunset.jpg") no-repeat fixed center;
+  color: #fcfafc;
+  background-size: cover;
+}
+
+/*Sections*/
+section {
+  height: auto;
+  padding:100px 100px;
+}
+
+.dark {
+  background-color:#0C0C0C;
+  color: #fcfafc;
+}
+
+.light {
+  background-color: #fcfafc;
+  color: #0C0C0C;
+}
+
+.blue {
+  background-color: #c7e8ff;
+  color: #0C0C0C;
+}
+
+/*Containers*/
+.container {
+  position: relative;
+  margin: 50px 50px;
+  text-align:center
+}
+
+/*Text Wrappers*/
+.abs-center {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  text-align: center;
+  -ms-transform: translate(-45%, -50%);
+  transform: translate(-50%, -50%);
+}
+
+.wrapper {
+  margin-right: auto; 
+  margin-left: auto; 
+  padding: 50px 10px 10px 10px;
+}
+
+.bio {
+  display: flex;
+  text-align: left;
+  max-width: 860px; 
+}
+
+.profile {
+  max-width: 1000px;
+  text-align: left;
+}
+
+.content {
+  text-align: left 
+}
+
+.intro {
+  margin: 0;
+  padding-left: 10px;
+}
+
+#fact-container {
   margin-top: 20px;
 }
+
+/*Cards*/
+.card {
+  background-color:#fff;
+  box-sizing: border-box;
+  border-radius: 15px;
+  overflow: hidden;
+  box-shadow: 0 5px 15px rgba(0,0,0,.2);
+  transition: .2s;
+  padding: 20px;
+  max-width: 33.33%;
```

Comment:
optional: calc(100%  / 3);

Reply:
had already deleted this line- will change for other 33.33s